manager/command.py

```diff
@@ -0,0 +1,44 @@
+from os import system
+
+
+class AbstractCommand:
```

Comment:
This class do nothing and still useless. Abstraction for abstraction, just eat memory, lines and programmer's attention =)